README.md

```diff
@@ -112,16 +116,53 @@ Write a conditional statement that...
 8.  Write your javascript solution below
 ```js
 // your answer here
+for(let a=0 ; a<=100 ;a++){
+
+if ( (a%3) == 0 ){
+console.log('Fizz')
+
+}
+else if ( (a%5) ==0) {
+console.log('Buzz')
+
+}
+else if ( (a%3 && a%5) == 0){
```

Comment:
This condition is not proper. it's the equivalent of "else if(a%5 == 0 || a%3 == 0)"
it will never run because the previous two conditions will always be true before this one.
a better way to do. think about this most special case.